Spigot-Server-Patches/0212-Add-a-warning-about-plugins-listening-on-the-synchro.patch

```diff
@@ -0,0 +1,54 @@
+From 6e4b069ce0cc3aea45bda60dd9b20081591ecd52 Mon Sep 17 00:00:00 2001
+From: Shane Freeder <theboyetronic@gmail.com>
+Date: Wed, 8 Feb 2017 02:41:31 +0000
+Subject: [PATCH] Add a warning about plugins listening on the synchronous
+ PlayerChatEvent
+
+
+diff --git a/src/main/java/com/destroystokyo/paper/PaperConfig.java b/src/main/java/com/destroystokyo/paper/PaperConfig.java
+index 7e0f6705..90a132c3 100644
+--- a/src/main/java/com/destroystokyo/paper/PaperConfig.java
++++ b/src/main/java/com/destroystokyo/paper/PaperConfig.java
+@@ -255,4 +255,9 @@ public class PaperConfig {
+         }
+         removeInvalidStatistics = getBoolean("settings.remove-invalid-statistics", false);
+     }
++
++    public static Boolean warnOnSyncChatListener = true;
```

Comment:
Actually, I don't think we need this either. It's not going to hurt people to see this in their logs. I don't want us to litter up configs with stuff like this.

We can just use a system property, so 
```java
if (System.getProperty("dontWarnOnSyncListeners") == null) { *warn* }
```